pkg/ipmi/ipmi.go

```diff
@@ -35,9 +36,20 @@ const (
 	_BMC_SET_WATCHDOG_TIMER = 0x24
 	_BMC_GET_WATCHDOG_TIMER = 0x25
 
+	_SET_SYSTEM_INFO_PARAMETERS = 0x58
```

Comment:
How about grouping all the commands (_BMC_SET_WATCHDOG_TIMER, _BMC_GET_WATCHDOG_TIMER and _SET_SYSTEM_INFO_PARAMETERS) together?

Reply:
Updated.